src/regressionmodel.jl

```diff
@@ -53,15 +48,20 @@ The approach for computing effect is based on the effects plots described here:
 Fox, John (2003). Effect Displays in R for Generalised Linear Models.
 Journal of Statistical Software. Vol. 8, No. 15
 """
+# TODO explain that can still use the transformation method themselves
```

Comment:
Inserting a comment between a method definition and its docstring breaks the link between the two:
```
julia> begin
       """
       hello
       """
       f(x) = 420
       end
f

help?> f
search: f fd for fma fld fld1 fill fdio frexp foldr foldl flush floor float first fill! fetch fldmod filter falses finally foreach fldmod1 findmin findmax findall

  hello

julia> begin
       """
       goodbye
       """
       # lol
       g(x) = 69
       end
g (generic function with 1 method)

help?> g
search: g get gcd GC get! gcdx gperm global getpid getkey gensym getfield getindex GlobalRef getproperty gethostname get_zero_subnormals @goto @gensym @generated

  No documentation found.

  g is a Function.

  # 1 method for generic function "g":
  [1] g(x) in Main at REPL[3]:6
```

Reply:
That TODO was TODO before actually reviewing :wink: